### DISCUSSION TITLE
User Stories

### DISCUSSION THREAD

**Discussion by vladrph** [Board-Kanban-template/Business-Analysis-Notes#11] [General]
As a dashboard web application user
I need to login 
So that input data into the web application dashboard. 
#5

**Comment by vladrph** [comment]
As a  dashboard web application user 
I need to quickly search for information  
So that I quickly view past notes and files . 
#12

**Comment by vladrph** [comment]
As a IMAG web application user  
I need view graphs and charts 
So that quickly view any trends and unusual patterns 
#13

**Comment by vladrph** [comment]
As a IMAG dashboard user
I need be able to send messages to others users 
So that communicate any issues about water quality to the appropriate  manager/user. 
#7

**Comment by vladrph** [comment]
As a IMAG dashboard application user 
I need analyze data and generate reports 
So that I can quickly view trends and historical water quality.
#15

**Comment by vladrph** [comment]
As a IMAG dashboard application user 
I need to securely input and store data. 
So that I can maintain the integrity of the water quality.
#10

**Comment by vladrph** [comment]
As a IMAG dashboard application user 
I need to be able to view notifications quickly 
So that I can quickly view any issues with water quality or needed information .
#14

**Comment by vladrph** [comment]
As a IMAG dashboard application user 
I need to be able to upload excel file  
So that I can quickly input historical water quality data  .
#9

**Comment by vladrph** [comment]
As a IMAG dashboard application user 
I need to be able access the dashboard from anywhere in the world   
So that I can quickly view and input any data needed.
#16

**Comment by vladrph** [comment]
As a IMAG dashboard application user 
I need to be able search for information quickly 
So that I can quickly view any data needed.
#17

**Comment by vladrph** [comment]
As a IMAG dashboard application user 
I need to be able access the dashboard with a tablet, phone, or desktop
So that I can quickly view and input any data needed.
#18

**Comment by vladrph** [comment]
As a IMAG dashboard application user 
I need to be able read the dashboard from anywhere in the world   
So that I can quickly view and input any data needed.
#19

**Comment by vladrph** [comment]
As a IMAG dashboard application user 
I need to be able to change my personal information. 
So that I can quickly view ensure that the dashboard has my most up to date contact information.
#6

**Comment by vladrph** [comment]
As a IMAG dashboard application user 
I need to be able to input water quality data online
So that I can view historical charts and water quality data. 
#8

**Comment by vladrph** [comment]
CON-1 The dashboard shall run on apple products. Software and hardware restrictions.  #20

**Comment by vladrph** [comment]
CON-2 #21 
  - Data interchange constraint. Historical can only be transmitted and uploaded via excel files. 
  - The user must be actively using google chrome, and the site should recommend it as the preferred browser.

**Comment by vladrph** [comment]
CON-3 #22 Restriction based on the web application's operating environment/ web browser

**Comment by tjmacphee** [comment]
CON-4 #23 The user must enable the accessibility features to make use of them (excluding tab indexes & screen readers for alt text)

**Comment by tjmacphee** [comment]
CON-5 #16 The hostee must pay the monthly fee to continue hosting the software for it to be live.

**Comment by tjmacphee** [comment]
CON-6 #14 The user must enable notifications through the browser, or email.

**Comment by tjmacphee** [comment]
As an IMAG user
I'd like to apply a colorblind filter & text-magnifier for the application
In order to enhance my experience.
#23

**Comment by tjmacphee** [comment]
As an IMAG user
I'd feel secure knowing my information is stored safely & information cannot be forged
So that I can trust in using this software
#24 & #25

**Comment by tjmacphee** [comment]
As an IMAG user
I'd like to know this app can function properly on google chrome
So I can avoid using another browser
#22
